### ISSUE TITLE
ATV Offroad Fury 3 (SCUS_974.05) erroneously identified as Genji: Dawn of the Samurai (Demo)

### ISSUE THREAD

**Issue by N1GHT-MAR3** [CosmicScale/PSBBN-Definitive-English-Patch#5]
SCUS_974.05 is incorrectly identified as a Genji: Dawn of the Samurai demo instead of ATV Offroad Fury 3. The game still works as normal, it just has the wrong name and cover art in PSBBN.

Looking at TitlesDB_PS2_English.csv, ATV3's serial is written as SLUS instead of SCUS. The Genji demo is then listed twice, first with ATV3's actual serial, then its own.

```
Line 172: SLUS_974.05|ATV Offroad Fury 3|Climax
Line 3404: SCUS_974.05|Genji: Dawn of the Samurai (Demo)|Climax
Line 3405: SCUS_975.04|Genji: Dawn of the Samurai (Demo)|Climax
```

![Screenshot 2024-12-16 05-27-22](https://github.com/user-attachments/assets/4338fc67-1b21-42b5-baca-3fa84b81a2cb)

**Comment by CosmicScale** [comment]
Thanks for reporting. Fixed in [Commit c0b164e](https://github.com/CosmicScale/PSBBN-Definitive-English-Patch/commit/c0b164e2525c5695bea19e37f0da8a81f00b9aee)

**Comment by GaryOak420** [comment]
I installed ATV Offroad Fury 3 tonight and it still uses the Genji thumbnail image.

**Comment by CosmicScale** [comment]
You need to delete the image from the art folder on your PC. The script will then download the correct image.

**Comment by CosmicScale** [comment]
Ooops, just checked and I only made the change to the TitlesDB_PS2_English.csv and forgot to change it in ArtDB.csv. Fixed it with commit https://github.com/CosmicScale/PSBBN-Definitive-English-Patch/commit/b345f70faa1ec66b10755b33a0f130e1f6772e15

You still need to delete the incorrect image from the icons/art folder, the artwork will be correct when you run the game installer again.

**Comment by GaryOak420** [comment]
Maybe I don't know where to look for this but I see no artwork files in any of my art folders on the PS2 HDD or my game library it syncs from.
 
 
  On Sun, Feb 16, 2025 at 5:46 AM, ***@***.***> wrote:    
Ooops, just checked and I only made the change to the TitlesDB_PS2_English.csv and forgot to change it in ArtDB.csv. Fixed it with commit b345f70

You still need to delete the incorrect image from the icons/art folder, the artwork will be correct when you run the game installer again.

—
Reply to this email directly, view it on GitHub, or unsubscribe.
You are receiving this because you commented.Message ID: ***@***.***>
CosmicScale left a comment (CosmicScale/PSBBN-Definitive-English-Patch#5)
Ooops, just checked and I only made the change to the TitlesDB_PS2_English.csv and forgot to change it in ArtDB.csv. Fixed it with commit b345f70

You still need to delete the incorrect image from the icons/art folder, the artwork will be correct when you run the game installer again.

**Comment by CosmicScale** [comment]
If you have already run the game installer script, in the `PSBBN-Definitive-English-Patch/icons/art` folder on your PC, you will find all the artwork that was downloaded. If the artwork for your game is not in there, when you run the game installer again the correct artwork will be downloaded.
